atlasaction/comments/schema-plan.tmpl

```diff
@@ -1,6 +1,13 @@
 <h3>Atlas detected changes to the desired schema</h3>
 <h4>Migration Plan {{- with .Plan.File.Link }} ({{- . | link "View on Atlas Cloud" -}}){{ end }}</h4>
-{{- .Plan.File.Migration | codeblock "sql" -}}
+
+{{/* Respect double new lines as they are skipped in <pre/code> blocks. */}}
+{{- with .Plan.File.Migration }}
```

Comment:
Please update the `codeblock` function instead

Reply:
I don't think so. This will affect all its usage. Instead, I prefer we use the markdown where needed (requires extra newlines), and creates unnecessary wrappers.